packages/office-addin-debugging/src/start.ts

```diff
@@ -173,41 +173,124 @@ export async function runPackager(commandLine: string, host: string = "localhost
 }
 
 /**
- * Start debugging
- * @param manifestPath The path to the manifest file.
- * @param appType The type of application to debug.
- * @param app The Office application to debug.
- * @param debuggingMethod The method to use when debugging.
- * @param sourceBundleUrlComponents Specify components of the source bundle url.
- * @param devServerCommandLine If provided, starts the dev server.
- * @param devServerPort If provided, port to verify that the dev server is running.
- * @param documentUrl If provided, the document to open for sideloading to web.
- * @param packagerCommandLine If provided, starts the packager.
- * @param packagerHost Specifies the host name of the packager.
- * @param packagerPort Specifies the port of the packager.
- * @param enableDebugging If false, start without debugging.
+ * `startDebugging` options.
```

Comment:
Consider removing the back ticks and making it `startDebugging() options` or `Options for startDebugging() function.`

Reply:
@wandyezj Let me know if you disagree with this, but I don't see a change here. Did you want to keep the back ticks?

Reply:
From what I've observed in other documentation it's standard to use backticks around the names of referred to functions or objects.

Reply:
Interesting. I haven't noticed this myself but would like to understand if this would be a better pattern to follow. Does this affect how it appears in VS Code IntelliSense?